### PR TITLE
Schema fixes, nodemon and module-aliases

### PR DESCRIPTION
**Pull request by isokosan**
**Fixes:**
- Schema** definition in parse-server options was mistyped as `schemas` instead of `schema`. Also the index on the `type` field on the `_User` schema was pointing to a missing field, causing an error.

**Features:**
- Added nodemon with module-aliases. A caveat here is that `tsc` needs to be run and compiled before starting the development server, for the module-aliases to find the relevant folders in the `dist` directory. Added `tsc` in the `dev` script to resolve this.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,15 +33,15 @@
...
         "@typescript-eslint/eslint-plugin": "^5.22.0",
         "@typescript-eslint/parser": "^5.22.0",
         "babel-eslint": "^10.1.0",
+        "babel-watch": "7.7.0",
         "eslint": "^8.15.0",
         "eslint-plugin-import": "^2.26.0",
         "jest": "^28.1.0",
+        "nodemon": "^2.0.16",
         "nyc": "^15.1.0",
-        "parse": "^3.4.2",
```
Comment (L3K0V) [inline package.json:32]: It's not needed?

**Review by azlekov — APPROVED** [review]
Do you think it's a good idea to include `@cloud`, `@parse`  `@schema` aliases?

File: package.json (hunk 1 above)
Comment (isokosan) [reply package.json:32]: `parse` is already required by `parse-server` so not needed. 

> Do you think it's a good idea to include `@cloud`, `@parse` `@schema` aliases?

You just need to put an extra slash when writing @/cloud` etc so I would say it is unnecessary.

**Comment by isokosan** [comment]
> Do you think it's a good idea to include `@cloud`, `@parse` `@schema` aliases?

You just need to put an extra slash when writing `@/cloud` etc so I would say it is unnecessary.

**Pull request merged**